types/custom-functions-runtime/index.d.ts

```diff
@@ -36,8 +56,19 @@ declare namespace CustomFunctions {
          * `{ "requiresAddress": true }`
          *
          * If the metadata JSON file is being generated from JSDoc comments, include the tag `@requiresAddress`.
+         * [Api set: CustomFunctionsRuntime 1.1]
          */
         address?: string;
+        /**
+         * The range addresses where the function parameters are located, if requested, otherwise undefined.
+         *
+         * To request the parameter addresses for the function, in the metadata JSON file, the function options should specify:
+         * `{ "requiresParameterAddresses": true }`
+         *
+         * If the metadata JSON file is being generated from JSDoc comments, include the tag `@requiresParameterAddresses`.
+         * [Api set: CustomFunctionsRuntime 1.1]
+         */
+        parameterAddresses?: string;
```

Comment:
Given that I don't have any context for this feature, it's not very clear to me what this would return when there are multiple parameters to the custom functions. A single custom function parameter could be a single cell address, or could represent a range as input, and there could be multiple parameters. When more than one, is this a list of ranges? It might help to explain this more in the JSDoc description. @mscharlock

Reply:
I want to check with Ruoying or Phoebe to make sure I write the right thing here but good call out